bears/vcs/git/GitCommitBear.py

```diff
@@ -291,6 +291,9 @@ def check_issue_reference(self, body,
         if host not in self.SUPPORTED_HOST_KEYWORD_REGEX:
             return
 
+        if not re.search(self.SUPPORTED_HOST_KEYWORD_REGEX[host], body):
+            return
```

Comment:
not necessary imo, the regex below just won't match and you achieve the same effect^^